ecoscope/analysis/classifier.py

```diff
@@ -0,0 +1,38 @@
+import mapclassify
+
+classification_methods = {
+    "equal_interval": mapclassify.EqualInterval,
+    "natural_breaks": mapclassify.NaturalBreaks,
+    "quantile": mapclassify.Quantiles,
+    "std_mean": mapclassify.StdMean,
+    "max_breaks": mapclassify.MaximumBreaks,
+    "fisher_jenks": mapclassify.FisherJenks,
+}
+
+
+# pass in a series and output the series
+def apply_classification(x, labels=None, scheme="natural_breaks", **kwargs):
+    """
+    Classifies the data in a GeoDataFrame column using specified mapclassify scheme.
+
+    Args:
+    y : An array containing the data to classify.
+    labels (str): labels of bins, use bin edges if labels==None.
+    scheme (str): Classification scheme to use. Must be one of the supported schemes by mapclassify.
```

Comment:
Let's keep this general:  "Classification scheme to use [equal_interval, natural_breaks, quantile, std_mean, max_breaks, fisher_jenks]"